components/banners/BannerContent.vue

```diff
@@ -0,0 +1,44 @@
+<template>
```

Comment:
This file should serve for the content layer for videos, banner images and callouts.